src/frontend/src/env/networks.env.ts

```diff
@@ -96,3 +96,23 @@ export const BTC_TESTNET_NETWORK: Network = {
 export const BITCOIN_NETWORKS: Network[] = [BTC_MAINNET_NETWORK, BTC_TESTNET_NETWORK];
 
 export const BITCOIN_NETWORKS_IDS: symbol[] = BITCOIN_NETWORKS.map(({ id }) => id);
+
+export const CHAIN_FUSION_MAINNET_NETWORK_SYMBOL = 'ChainFusion';
+
+export const CHAIN_FUSION_MAINNET_NETWORK_ID = Symbol(CHAIN_FUSION_MAINNET_NETWORK_SYMBOL);
+
+export const CHAIN_FUSION_MAINNET_NETWORK: Network = {
+	id: CHAIN_FUSION_MAINNET_NETWORK_ID,
+	env: 'mainnet',
+	name: 'Chain Fusion'
+};
+
+// TODO: remove this when we have finished implementing the single-page view with Chain Fusion
+export const CHAIN_FUSION_ENABLED =
+	JSON.parse(import.meta.env.VITE_CHAIN_FUSION_ENABLED ?? false) === true;
```

Comment:
Thinking at it now, should we actually be more verbose to ensure it's related to the network? i.e. `NETWORK_CHAIN_FUSION` instead of `CHAIN_FUSION`? 

Both contants and env variable. WDYT?

Reply:
agreed!